packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -85,7 +132,7 @@ const Statement = ({
                 <div className='statement__account-statistics-item'>
                     <div className='statement__account-statistics--is-rectangle'>
                         <span className='statement__account-statistics-title'>
-                            {localize('Total deposits')} {`(${currency})`}
+                            {localize('Total deposits')} {is_mobile && `(${currency})`}
```

Comment:
Please use `isMobile` from `@deriv/shared` for consistency

But you can use `MobileWrapper` as well, since this is a mobile only rendered element